### DISCUSSION TITLE
怎么在scss文件中使用 designToken

### DISCUSSION THREAD

**Discussion by Half-Sugar-Code** [ant-design/ant-design#41220] [Q&A]
hello，我现在有一个问题
```
      theme={{
        token: {
          colorPrimary: "#00b96b",
        },
      }}
```
我使用这种方式指定themeToken，在组件内部使用token提供的颜色的时候，可以以行内样式的方式使用,我需要怎么配置才可以在`scss`中使用指定的token呢？
```
const { defaultAlgorithm, defaultSeed } = theme;
const mapToken = defaultAlgorithm(defaultSeed);

style={{
     color: mapToken.bule;
}}
```

**Comment by zombieJ** [comment]
[静态消费（如 less）](https://ant.design/docs/react/customize-theme-cn#%E9%9D%99%E6%80%81%E6%B6%88%E8%B4%B9%E5%A6%82-less)

**Comment by Half-Sugar-Code** [comment]
官网描述了如何配置less-loader，没有使用的demo，不是很清楚具体应该如何操作一下

**Comment by dealdot** [comment]
less中使用 modifyVars: mapToken注入到less中可直接使用，但这个值是固定的，切换暗黑和亮色主题的时候值是不会变的，还是得使用useToken
